CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+* Perform actions when aqua scan detects security vulnerabilities with solutions ([#1147](https://github.com/opendevstack/ods-jenkins-shared-library/pull/1147))
```

Comment:
Actually the code is more specific. It's about remotely exploitable vulnerabilities, not about any possible fix.
Please be precise for a changelog reader.